packages/components/src/ui/scrollable/test/__snapshots__/index.js.snap

```diff
@@ -78,14 +78,14 @@ exports[`props should render smoothScroll 1`] = `
 - + Second value
```

Comment:
Any Idea why this snapshot prints twice:
```
Snapshot Diff:
 - First value
 + Second value
```

There is more outputs like that.

Reply:
Oh that's odd 🤔 I'll do some poking around, thanks for pointing this out.